docs/src/reference/workflow-config-file.rst

```diff
@@ -200,7 +200,7 @@ subsampling
 -----------
 
 -  type: object
--  description: Schemes for subsampling data prior to phylogenetic inference to avoid sampling bias or focus an analysis on specific spatial and/or temporal scales. `See the SARS-CoV-2 tutorial for more details on defining subsampling schemes <../reference/customizing-analysis.html#subsampling>`__.
```

Comment:
Rather than removing this sentence, it should be replaced with another example that exists. Maybe something like this:

> See the [genomic surveillance tutorial](https://docs.nextstrain.org/projects/ncov/en/latest/tutorial/genomic-surveillance.html) for an example.

Reply:
fixed.